src/main/java/com/atlassian/db/replica/api/AuroraPostgresLsnReplicaConsistency.java

```diff
@@ -69,14 +69,14 @@ public boolean isConsistent(Supplier<Connection> replica) {
      * @return LSN (log sequence number) for the main database (the highest LSN)
      */
     private long queryMainDbLsn(Connection connection) {
-        return queryLsn(connection, "SELECT MAX(durable_lsn) AS lsn FROM aurora_global_db_instance_status();");
+        return queryLsn(connection, "SELECT MAX(durable_lsn) AS lsn FROM aurora_replica_status();");
```

Comment:
Should we use compatibility mode?

Reply:
It's not necessary - it is compatible with previous one.

Reply:
compatibility mode should be used with a reference to the working version. The previous one didn't worked, right?

Reply:
IMO compatibility mode is to keep possibility to dynamically change implementation on production between two "incompatible" versions. This change is compatible with previous one, so it's unnecessary to use this mode.

Reply:
To clarify. `compatibility mode` is to FF any change, especially changes like this that can't be FF the other way (we don't introduce a new API to opt-in, we change the behaviour of the previous one).

Reply:
@wyrzyk indeed. Let's not try to define a "working version".

Reply:
Agree